test/e2e/run_e2e.sh

```diff
@@ -30,6 +32,17 @@ export ETCD_ENDPOINTS="$ETCD_ENDPOINTS"
 export CLUSTER_ID="$CLUSTER_ID"
 export WORKER_SHARED_SECRET="$WORKER_SHARED_SECRET"
 
+if [[ "${E2E_NODE_OS}" == "coreos" ]]; then
+    export MASTER_PROFILE="master"
+    export WORKER_PROFILE="worker"
+elif [[ "${E2E_NODE_OS}" == "ubuntu" ]]; then
+    export MASTER_PROFILE="master-ubuntu"
+    export WORKER_PROFILE="worker-ubuntu"
+else
+    echo "Unsupported E2E_NODE_OS: ${E2E_NODE_OS}"
+    exit 1
+fi
```

Comment:
Just as an idea: The cluster YAML for e2e tests above is already a list of clusters (but contains only one cluster at the moment). Rather than templating the config like this we could also add a new entry to the list to say "please run the e2e tests against the following clusters".

Reply:
I thought about that but this way we can see each cluster as it's own CDP step.